include/miral/miral/window_manager_tools.h

```diff
@@ -54,10 +54,11 @@ class Workspace;
 class WindowManagerToolsImplementation;
 
 /// Window management functions for querying and updating MirAL's model
+/// \remark Since MirAL 3.0
 class WindowManagerTools
 {
 public:
-    explicit WindowManagerTools(WindowManagerToolsImplementation* tools);
+    explicit WindowManagerTools(miral::WindowManagerToolsImplementation* tools);
```

Comment:
What's the failure mode if the (otherwise unnecessary) `miral::` namespace specifier is left off here?

Reply:
Explained in comment